rpcserver.go

```diff
@@ -1337,6 +1337,7 @@ func (r *rpcServer) GetTransactions(context.Context,
 			Amount:           tx.Value.ToBTC(),
```

Comment:
I was wondering if satoshis (int64) should be used for the amount instead of BTC (float64)?